cilium bpf sha get ... CLI issues

### Issue 1

```
# cilium bpf sha get
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/cilium/cilium/cilium/cmd.glob..func18(0x2f95180, 0x3551240, 0x0, 0x0)
        /go/src/github.com/cilium/cilium/cilium/cmd/bpf_sha_get.go:34 +0x69
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).execute(0x2f95180, 0x3551240, 0x0, 0x0, 0x2f95180, 0x3551240)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:766 +0x2ae
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2f90ca0, 0xc0002c1f68, 0x16649ee, 0x2f9d8e0)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:850 +0x2fc
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).Execute(...)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:800
github.com/cilium/cilium/cilium/cmd.Execute()
        /go/src/github.com/cilium/cilium/cilium/cmd/root.go:47 +0x32
main.main()
        /go/src/github.com/cilium/cilium/cilium/main.go:22 +0x20
```

This should print the CLI usage instead of a stack trace.

### Issue 2

```
# cilium bpf sha get dfe53981733b0f602689eb9c04fcc1259578a32c -o json
"#include \"lib/utils.h\"\n\nDEFINE_IPV6(LXC_IP, 0x20, 0x1, 0xdb, 0x8, 0xb, 0xad, 0xca, 0xfe, 0x60, 0xd, 0xbe, 0xe2, 0xb, 0xad, 0xca, 0xfe);\nDEFINE_U32(LXC_IPV4, 0x030200c0);\t/* 50462912 */\n#define LXC_IPV4 fetch_u32(LXC_IPV4)\nDEFINE_MAC(NODE_MAC, 0x2, 0x0, 0x60, 0xd, 0xf0, 0xd);\n#define NODE_MAC fetch_mac(NODE_MAC)\nDEFINE_U32(LXC_ID, 0x0000ffff);\t/* 65535 */\n#define LXC_ID fetch_u32(LXC_ID)\nDEFINE_U32(SECLABEL, 0x00000002);\t/* 2 */\n#define SECLABEL fetch_u32(SECLABEL)\nDEFINE_U32(SECLABEL_NB, 0x02000000);\t/* 33554432 */\n#define SECLABEL_NB fetch_u32(SECLABEL_NB)\n#define POLICY_MAP cilium_policy_65535\n#define CALLS_MAP cilium_calls_65535\n#define CONFIG_MAP cilium_ep_config_65535\n#define FORCE_LOCAL_POLICY_EVAL_AT_SOURCE 1\n#define ENABLE_ROUTING 1\n#define ENABLE_ARP_RESPONDER 1\n#define ENABLE_HOST_REDIRECT 1\n#define CT_MAP_TCP4 cilium_ct4_global\n#define CT_MAP_ANY4 cilium_ct_any4_global\n#define CT_MAP_TCP6 cilium_ct6_global\n#define CT_MAP_ANY6 cilium_ct_any6_global\n#define CT_MAP_SIZE_TCP 524288\n#define CT_MAP_SIZE_ANY 262144\n#define LB_L3 1\n#define LB_L4 1\n#define LOCAL_DELIVERY_METRICS 1\n#define CONNTRACK 1\n#define CONNTRACK_ACCOUNTING 1\n#undef CONNTRACK_LOCAL\n#undef DEBUG\n#undef LB_DEBUG\n#define DROP_NOTIFY 1\n#define MONITOR_AGGREGATION 3\n#undef ENABLE_NAT46\n#define TRACE_NOTIFY 1\n"
```

Maybe in some form, this is kind-of json? Because it's just a string. However I'd expect something like this instead:

```
# cilium bpf sha get <sha> -o json
{
  "LXC_IP": "2001:db08:0bad:cafe:600d:bee2:0bad:cafe",
  "LXC_IPV4": 10.0.2.3",
  ...
}
```

--- a/cilium/cmd/bpf_sha_get.go
+++ b/cilium/cmd/bpf_sha_get.go
@@ -31,7 +31,11 @@ var bpfShaGetCmd = &cobra.Command{
 	Short:   "Get datapath SHA header",
 	Run: func(cmd *cobra.Command, args []string) {
 		common.RequireRootPrivilege("cilium bpf sha get")
-		dumpSha(args[0])
+		if len(args) == 0 {
+			cmd.Help()
+		} else {
+			dumpSha(args[0])
+		}
 	},
 }
 

